src/Particular.PlatformSample/PlatformLauncher.cs

```diff
@@ -24,7 +24,8 @@ public static class PlatformLauncher
         /// message transport.
         /// </summary>
         /// <param name="showPlatformToolConsoleOutput">By default the output of each application is suppressed. Set to true to show tool output in the console window.</param>
-        public static void Launch(bool showPlatformToolConsoleOutput = false)
+        /// <param name="servicePulseDefaultRoute">By default the service pulse dashboard (/dashboard) is displayed. Set the default route to any valid service pulse route such as (/monitored_endpoints).</param>
+        public static void Launch(bool showPlatformToolConsoleOutput = false, string servicePulseDefaultRoute = null)
```

Comment:
Why not just have the default value of `servicePulseDefaultRoute` be `/dashboard`?